docs/supported_languages_and_lockfiles.md

```diff
@@ -22,19 +22,19 @@ nav_order: 2
-| :--------- | :----------------------------------------------------------------------------------------------------------------------- |
-| C/C++      | `conan.lock`<br>[C/C++ commit scanning](#cc-scanning)                                                                    |
-| Dart       | `pubspec.lock`                                                                                                           |
-| Elixir     | `mix.lock`                                                                                                               |
-| Go         | `go.mod`                                                                                                                 |
-| Java       | `buildscript-gradle.lockfile`<br>`gradle.lockfile`<br>`pom.xml`[\*](https://github.com/google/osv-scanner/issues/35)     |
-| Javascript | `package-lock.json`<br>`pnpm-lock.yaml`<br>`yarn.lock`                                                                   |
-| PHP        | `composer.lock`                                                                                                          |
-| Python     | `Pipfile.lock`<br>`poetry.lock`<br>`requirements.txt`[\*](https://github.com/google/osv-scanner/issues/34)<br>`pdm.lock` |
-| R          | `renv.lock`                                                                                                              |
-| Ruby       | `Gemfile.lock`                                                                                                           |
-| Rust       | `Cargo.lock`                                                                                                             |
+| Language   | Compatible Lockfile(s)                                                                                                                                      |
+| :--------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| C/C++      | `conan.lock`<br>[C/C++ commit scanning](#cc-scanning)                                                                                                       |
+| Dart       | `pubspec.lock`                                                                                                                                              |
+| Elixir     | `mix.lock`                                                                                                                                                  |
+| Go         | `go.mod`                                                                                                                                                    |
+| Java       | `buildscript-gradle.lockfile`<br>`gradle.lockfile`<br>`pom.xml`[\*](https://github.com/google/osv-scanner/issues/35),<br>`gradle/verification-metadata.xml` |
```

Comment:
should this be alphabetical order?

Reply:
oh yeah good point - I originally thought the `*` was applying to all of these but its actually just `pom.xml` so we could sort it alphabetically.